kernel/src/main/java/org/kframework/kprove/KProveOptions.java

```diff
@@ -55,6 +56,13 @@ public synchronized File specFile(FileUtil files) {
     @Parameter(names="--depth", description="The maximum number of computational steps to prove")
     public Integer depth;
 
+    @Parameter(names="--concrete-rules", description="List of concrete rule labels")
```

Comment:
Description should be expanded. I guess you mean:
```
List of rule labels to be considered concrete, in addition to rules marked with `[concrete]` attribute.
```

Reply:
Another concern: Option should be moved to `JavaExecutionOptions`. Then it will be available to other tools like KEq, and only in Java backend.